src/struct/Searcher.js

```diff
@@ -37,13 +39,42 @@ const Searcher = class Searcher {
         response.on('data', chunk => chunks.push(chunk));
         return response.on('end', () => res(JSON.parse(Buffer.concat(chunks).toString())[0]));
       });
+      req.end();
+    });
+  }
+
+  bulkLookup(ids) {
+    if (!Array.isArray(ids) || ids.length < 2) {
+      return Promise.reject(new Error('You must define an array with 2 or more ID\'s defined.'));
+    }
+    let validIDs = ids.filter(id => regex.test(id));
+    if (validIDs.length > 99) {
+      return Promise.reject(new Error(`More than 99 ID's for bulk lookup are not allowed!`));
+    }
+    const options = Object.assign({}, option_store.get(this));
+    let path = `?user_id=${validIDs[0]}`;
+    for (let id of validIDs) {
+      path += `&user_id=${id}`;
+    }
+    options.path += path;
+    return new Promise((res, rej) => {
+      const req = https.get(options, response => {
+        const { statusCode } = response;
+        if (statusCode !== 200) {
+          rej(new Error(`Request failed. Status Code: ${statusCode}`));
+          return response.resume();
+        }
+        let chunks = [];
+        response.on('data', chunk => chunks.push(chunk));
+        return response.on('end', () => res(JSON.parse(Buffer.concat(chunks).toString())));
+      });
       return req.end();
     });
   }
 
   async isBanned(id) {
-    if (!/^\d+$/.test(id)) throw new Error('Discord User Snowflake ID must contain only numbers.');
-    return !!+(await this.lookup(id)).banned;
+    if (!regex.test(id)) throw new Error('Discord User Snowflake ID must contain only numbers.');
+    return !!(await this.lookup(id)).banned;
```

Comment:
Replacing `return !!+(await this.lookup(id)).banned;` with `return !!(await this.lookup(id)).banned;` causes an issue as the value returned by the API is a string containing 0 or 1 instead of an expected number or boolean, which is why I prepend the value with `+`. !!+'0' becomes false properly, whereas !!'0' is actually a truth value.